common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -1778,8 +1784,11 @@ def test_response_types_multiple_tags(self):
         self.assertEquals(
             descriptor.index_dictionary(), {
                 'content_type': CapaDescriptor.INDEX_CONTENT_TYPE,
-                'display_name': name,
-                'problem_types': ["optionresponse", "multiplechoiceresponse"]
+                'problem_types': ["optionresponse", "multiplechoiceresponse"],
+                'content': {
+                    'display_name': name,
+                    'capa_content': ' Label Some comment Donut Buggy '
+                }
```

Comment:
It doesn't seem like all the data intended to be discarded by the regex above is tested in these tests. All those conditions - and perhaps combinations of those conditions - should be tested.